scripts/edit-distributor-claim-windows.ts

```diff
@@ -10,9 +10,9 @@ const CX = new Connection(
   "confirmed"
 );
 
-const CLAIM_START_TS = new anchor.BN(1717286400); // June 2 00:00
-const FULL_CLAIM_FROM = new anchor.BN(1717891200); // June 9 00:00
-const CLAIM_END_TS = new anchor.BN(1718409600); // June 15 00:00
+const CLAIM_START_TS = new anchor.BN(1718961368); // June 2 00:00
```

Comment:
idk what these are but the commented times are wrong now

Reply:
ah yes ill fix it later its not that important